logic/src/androidMain/kotlin/com/wire/kalium/logic/CoreLogic.kt

```diff
@@ -30,7 +31,10 @@ actual class CoreLogic(
                 backendConfig = serverConfigMapper.toBackendConfig(serverConfig = session.serverConfig),
                 isRequestLoggingEnabled = BuildConfig.DEBUG //TODO: Multi-platform logging solution!
             )
+
             val proteusClient: ProteusClient = ProteusClientImpl(rootProteusDirectoryPath, session.userId)
+            runBlocking { proteusClient.open() }
```

Comment:
I don't quite remember what's the importance of closing Cryptobox 😞 

I'd like to gather @typfel's experiene here.

Can't we just do this on `ProteusClientImpl$init`?

```kotlin
actual class ProteusClientImpl(constructorStuff){
    
    init {
        runBlocking { open() }
    }
    // ...
}

Reply:
yep we totally can, the issue here is can we do the same on JS and and IOS since open() is a suspend function (it need to be suspend only because of JS)

Reply:
Yeah, @MohamadJaara and I discussed this yesterday. We need to revisit this and potentially implicitly `open()` in the init but then also revise the API so that it's compatible with JS where most things are async.

The importance of closing cryptobox would potentially be to free up any locks which might be taken. But they might anyway be released when he application terminates. I think it's even more important guarantee that we only ever open one cryptobox per session.